seedlet-blocks/theme.json

```diff
@@ -434,7 +434,7 @@
 			},
 			"core/site-title": {
 				"typography": {
-					"fontSize": "60px",
+					"fontSize": "var(--wp--preset--font-size--normal)",
```

Comment:
I believe this causes a visual regression in the header: 

<img width="765" alt="Screen Shot 2021-05-27 at 10 30 40 AM" src="https://user-images.githubusercontent.com/5375500/119844645-779ff480-bebd-11eb-87b9-e100d26020f3.png">

We can address it by setting the font size of the site title in the header template.

Reply:
Done here https://github.com/Automattic/themes/pull/3956/commits/c0e853e9201cc34920f3c9a63afafd3a5276a699